build.sbt

```diff
@@ -132,6 +132,14 @@ lazy val interfaces = project
     libraryDependencies ++= List(
       "io.get-coursier" % "interface" % V.coursier
     ),
+    // @note needed to deal with issues with dottyDoc
```

Comment:
So I know we've done this in various repos now. Originally I thought it had to do with an older version of Dotty 0.27 and sbt. What is the status on the actual core issue of this? Is this fixed if we were only building for the newer Scala 3 version with the latest sbt?

Reply:
I think every time is a new issues :sweat_smile: 

I am talking with the team to figure it out.

Reply:
The problem here is that `interfaces` is a Java-only project and scala3doc fails in such a case. We are working on the fix.